### PR TITLE
More updates for pi 5 2gb and some pico 2 updates

### PR DESCRIPTION
**Pull request by nathan-contino**

### CODE REVIEW

**Comment by nathan-contino** [comment]
@timg236 can you confirm or deny this revision code for pi 5 2GB?

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc [hunk 1]
```diff
@@ -610,6 +610,12 @@ NOTE: This list is not exhaustive - there may be codes in use that are not in th
 | 512MB
 | Sony UK
 
+| b04170
```
Comment (JamesH65) [inline documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc:613]: I think this should be b04171, it's a revision 1.1 board. But I don't have one to check.

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc [hunk 2]
```diff
@@ -610,6 +610,12 @@ NOTE: This list is not exhaustive - there may be codes in use that are not in th
...
+| 5
+| 1.0
```
Comment (JamesH65) [inline documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc:615]: 1.1, see above.

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc (hunk 1 above)
Comment (timg236) [reply documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc:613]: It's revision 1.0 - revision 1.1 boards might follow

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc (hunk 2 above)
Comment (nathan-contino) [reply documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc:615]:
```suggestion
| 1.1
```

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc (hunk 1 above)
Comment (nathan-contino) [reply documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc:613]:
```suggestion
| b04171
```

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc (hunk 2 above)
Comment (nathan-contino) [reply documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc]: Looks like Tim confirmed this is an R0 1.0 board, so I'll switch this back. Read the comments in the wrong order.

File: documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc [hunk 3]
```diff
@@ -610,6 +610,12 @@ NOTE: This list is not exhaustive - there may be codes in use that are not in th
 | 512MB
 | Sony UK
 
+| b04170
+| 5
+| 1.1
```
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc:615]:
```suggestion
| 1.0
```

**Pull request merged**